pkgs/applications/networking/remote/aws-workspaces/default.nix

```
@@ -5,15 +5,15 @@
 
 stdenv.mkDerivation rec {
   pname = "aws-workspaces";
-  version = "4.1.0.1523";
+  version = "4.5.0.2006";
 
   src = fetchurl {
-    # ref https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/bionic/main/binary-amd64/Packages
+    # ref https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/focal/main/binary-amd64/Packages
     urls = [
-      "https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/bionic/main/binary-amd64/workspacesclient_${version}_amd64.deb"
-      "https://web.archive.org/web/20220709124028/https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/bionic/main/binary-amd64/workspacesclient_${version}_amd64.deb"
+      "https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/focal/main/binary-amd64/workspacesclient_${version}_amd64.deb"
+      "https://web.archive.org/web/20220709124028/https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/focal/main/binary-amd64/workspacesclient_${version}_amd64.deb"
```

I'm getting a 404 for https://web.archive.org/web/20220709124028/https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/focal/main/binary-amd64/workspacesclient_4.5.0.2006_amd64.deb , did you upload the deb file to archive.org?

---

I did not. I figured that might be already archived since the other one is.

---

> None of the checkboxes are ticked in the PR description... presumably you've tested this on your machine at least yeah? slightly_smiling_face

This is still a 'draft' since it's work in progress and I'm still trying to get it to build. Perhaps you could help me with it since you're here though? I'm new to NixOS.

---

It is now captured, I chose the option on their site to crawl it.

---

Unfortunately it has to be uploaded manually. I just did that (I don't know why the URL is different now :man_shrugging: )

```suggestion
      "https://archive.org/download/workspacesclient_4.5.0.2006_amd64/workspacesclient_4.5.0.2006_amd64.deb"
```

---

https://archive.org/upload